lib/ueberauth/strategy/microsoft/oauth.ex

```diff
@@ -42,14 +49,21 @@ defmodule Ueberauth.Strategy.Microsoft.OAuth do
     |> AuthCode.get_token(params, headers)
   end
 
+  def refresh_token(client, params, headers) do
+    client
+    |> put_param(:client_id, client.client_id)
+    |> put_param(:client_secret, client.client_secret)
+    |> Refresh.get_token(params, headers)
+  end
+
   defp defaults(config) do
     tenant_id = config[:tenant_id] || "common"
 
     [
       strategy: __MODULE__,
-      site: "https://graph.microsoft.com",
+      site: "https://login.microsoftonline.com/#{tenant_id}/oauth2/v2.0",
       authorize_url: "https://login.microsoftonline.com/#{tenant_id}/oauth2/v2.0/authorize",
-      token_url: "https://login.microsoftonline.com/#{tenant_id}/oauth2/v2.0/token",
+      token_url: "/token",
```

Comment:
I wasn't able to get authentication working when only using `/token`. Am I missing something here?